theseus/src/state/profiles.rs

```diff
@@ -254,13 +267,15 @@ impl Profiles {
         self.insert(Self::read_profile_from_dir(&path.canonicalize()?).await?)
     }
 
+    // Remove a profile.
     #[tracing::instrument(skip(self))]
     pub fn remove(&mut self, path: &Path) -> crate::Result<&Self> {
         let path = PathBuf::from(path.canonicalize()?.to_str().unwrap());
```

Comment:
Also noticed an unwrap slipped in here. Should just not remove the path if it doesn't exist instead of crashing the program haha

Reply:
Oops lol, fixed